forte/api/forte_python_module.cc

```diff
@@ -256,6 +259,8 @@ PYBIND11_MODULE(forte, m) {
              "Return the DSRG dressed dipole integrals")
         .def("nuclear_dipole", &MASTER_DSRG::nuclear_dipole,
              "Return nuclear components of dipole moments")
+        .def("Hbar0", &MASTER_DSRG::Hbar0,
```

Comment:
Hbar0 is just the expectation value of the reference wavefunction, yes? I think it's worth mentioning that here, for those who aren't in the weeds of DSRG theory.

Reply:
Done!